ts/src/renderer/phaser/classes/PhaserProjectile.ts

```diff
@@ -12,37 +15,44 @@ class PhaserProjectile extends Phaser.GameObjects.Container {
 		const key = `projectile/${projectile._stats.type}`;
 
 		const sprite = this.sprite = scene.add.sprite(0, 0, key);
+		const translate = projectile._translate;
+		const bounds = projectile._bounds2d;
+		this.x = translate.x;
```

Comment:
we can use `setPosition` here, similar to how it is done below with `setDisplaySize`

Reply:
which method is this `setPositon()`? The only reference I'm finding is a box2d method

Reply:
nvm I found it. As expected there was no reference in the repo because it belongs to the Phaser source. Cool.